x-pack/legacy/plugins/monitoring/common/constants.js

```diff
@@ -181,3 +178,47 @@ export const CODE_PATH_LOGSTASH = 'logstash';
 export const CODE_PATH_APM = 'apm';
 export const CODE_PATH_LICENSE = 'license';
 export const CODE_PATH_LOGS = 'logs';
+
+/**
+ * The header sent by telemetry service when hitting Elasticsearch to identify query source
+ * @type {string}
+ */
+export const TELEMETRY_QUERY_SOURCE = 'TELEMETRY';
+
+/**
+ * The name of the Kibana System ID used to publish and look up Kibana stats through the Monitoring system.
+ * @type {string}
+ */
+export const KIBANA_SYSTEM_ID = 'kibana';
+
+/**
+ * The name of the Beats System ID used to publish and look up Beats stats through the Monitoring system.
+ * @type {string}
+ */
+export const BEATS_SYSTEM_ID = 'beats';
+
+/**
+ * The name of the Apm System ID used to publish and look up Apm stats through the Monitoring system.
+ * @type {string}
+ */
+export const APM_SYSTEM_ID = 'beats';
```

Comment:
@chrisronline in telemetry we were using `beats` as `APM_SYSTEM_ID` but it seems you've added it as `apm` in your latest PR.

Reply:
okay so the only place we are using this constant is here:
```
case BEATS_SYSTEM_ID:
case APM_SYSTEM_ID:
      return INDEX_PATTERN_BEATS;
```

so having `APM_SYSTEM_ID = 'beats'` is techinically like writing

```
case 'beats':
case 'beats':
    return INDEX_PATTERN_BEATS;
```

I've kept your change and it works fine. I have checked the payload after merging your change to my PR however and it seems that we no longer recieve `beats` usage data.

We collect beats usage data by querying the following index patterns: `'.monitoring-beats-6-*,.monitoring-beats-7-*'`

and then we do some processing, the code lives here:
x-pack/legacy/plugins/monitoring/server/telemetry_collection/get_beats_stats.js

I believe with the introduced changes on master something needs to change in he collecion function?

Reply:
I'm not following this logic unfortunately. 

I'm only seeing `APM_SYSTEM_ID` referenced in non-telemetry code.

I brought down this PR and am able to see `beats` usage data (assuming I have a running beat with monitoring enabled) as expected (by manually curling `http://localhost:5601/api/telemetry/v2/clusters/_stats`).

Can you provide some repo steps and I can take a look?

Reply:
Ah yes i've ran a beat and indeed we do get that data in telemetry. I think everything is good here now, thanks!